bibtex-actions.el

```diff
@@ -39,6 +39,8 @@
 
 (require 'bibtex-completion)
 
+(declare-function org-element-context "org-element")
+
```

Comment:
```suggestion
(declare-function org-element-property "org-element")

```

Declaring `org-element-property`, which is also used below, should eliminate the warning